libs/template/templates/default-python/template/{{.project_name}}/databricks.yml.tmpl

```diff
@@ -0,0 +1,32 @@
+bundle:
+  name: com.mycompany.{{.project_name}}
+
+include:
+  - res/*.yml
+
+artifacts:
+    {{.project_name}}:
+      type: whl
+      path: "."
+      build: "python3 setup.py bdist_wheel"
+
+environments:
+  dev:
```

Comment:
In other places, we have used `development`. Either is fine as long as we are consistent